app/scripts/modules/amazon/src/loadBalancer/AmazonLoadBalancersTag.tsx

```diff
@@ -99,66 +99,77 @@ export class AmazonLoadBalancersTag extends React.Component<ILoadBalancersTagPro
-            <Tooltip value={`${showPopover ? 'Hide' : 'Show'} all ${totalCount} ${showValue}`}>
-              <button
-                className="btn btn-link btn-multiple-load-balancers clearfix no-padding"
-                onClick={this.toggleShowPopover}
-              >
-                <span className="badge badge-counter">
-                  <span className="icon"><span className="icon-elb"/></span> {totalCount}
-                </span>
-              </button>
-            </Tooltip>
-          )}
-          { showPopover && (
-            <div className="menu-load-balancers">
-              {loadBalancerCount > 0 && <div className="menu-load-balancers-header">Load Balancers</div>}
-              {sortBy(loadBalancers, 'name').map((loadBalancer) => <LoadBalancerListItem key={loadBalancer.name} loadBalancer={loadBalancer} onItemClick={this.showLoadBalancerDetails}/>)}
-              {targetGroupCount > 0 && <div className="menu-load-balancers-header">Target Groups</div>}
-              {sortBy(targetGroups, 'name').map((targetGroup) => <LoadBalancerListItem key={targetGroup.name} loadBalancer={targetGroup} onItemClick={this.showTargetGroupDetails}/>)}
-            </div>
-          )}
-          { (loadBalancers.length === 1 && targetGroups.length === 0) && (
-            <span className="btn-load-balancer">
-              <LoadBalancerButton key={loadBalancers[0].name} label="Load Balancer" loadBalancer={loadBalancers[0]} onItemClick={this.showLoadBalancerDetails}/>
-            </span>
-          )}
-          { (targetGroups.length === 1 && loadBalancers.length === 0) && (
-            <span className="btn-load-balancer">
-              <LoadBalancerButton key={targetGroups[0].name} label="Target Group" loadBalancer={targetGroups[0]} onItemClick={this.showTargetGroupDetails}/>
-            </span>
-          )}
-        </span>
-      )
-    } else {
+    if (!totalCount) {
       return null;
     }
+
+    const className = `load-balancers-tag ${totalCount > 1 ? 'overflowing' : ''}`;
+    const popover = (
+      <div className="menu-load-balancers">
+        {loadBalancerCount > 0 && <div className="menu-load-balancers-header">Load Balancers</div>}
+        {sortBy(loadBalancers, 'name').map((loadBalancer) => (
+          <LoadBalancerListItem key={loadBalancer.name} loadBalancer={loadBalancer} onItemClick={this.showLoadBalancerDetails}/>
+        ))}
+
+        {targetGroupCount > 0 && <div className="menu-load-balancers-header">Target Groups</div>}
+        {sortBy(targetGroups, 'name').map((targetGroup) => (
+          <LoadBalancerListItem key={targetGroup.name} loadBalancer={targetGroup} onItemClick={this.showTargetGroupDetails}/>
+        ))}
+      </div>
+    );
+
+    return (
+      <span className={className}>
+        { totalCount > 1 && (
+          <HoverablePopover
+            delayShow={100}
+            delayHide={150}
+            onShow={this.handleShowPopover}
+            placement="bottom"
```

Comment:
I should steal our code from UI Bootstrap to allow us to do `auto` or even better `auto bottom`.